target/snitch_cluster/Makefile

```diff
@@ -53,7 +53,7 @@ VLT_AR = ${VLT_BUILDDIR}/Vtestharness__ALL.a
 # (LRU) config, all targets depending on the configuration file have
 # to be rebuilt. This file is used to express this condition as a
 # prerequisite for other rules.
-DEFAULT_CFG = cfg/default.hjson
+DEFAULT_CFG = cfg/divsqrt.hjson
```

Comment:
I am not sure if we want to enable the Div/Sqrt unit by default

Reply:
@colluca leaving this to you.

Reply:
True, the default should remain `default.hjson`. I will fix this.